api/clients/v2/cert_verifier_address_provider.go

```diff
@@ -0,0 +1,10 @@
+package clients
+
+// CertVerifierAddressProvider defines an object which can translate block number to cert verifier address
+//
+// This provider uses block number as a key, since updates to a cert verifier address in a running system are
+// coordinated by defining the block number at which a new cert verifier address takes effect.
+type CertVerifierAddressProvider interface {
+	// GetCertVerifierAddress returns the EigenDACertVerifierAddress that is active at the input block number
+	GetCertVerifierAddress(blockNumber uint64) (string, error)
```

Comment:
```suggestion
	GetCertVerifierAddress(blockNumber uint64) (common.Address, error)
```

Reply:
would using geth's Address type make sense here? Typically prefer this over a string

Reply:
Yeah, I think that makes sense. [a01d8fed](https://github.com/Layr-Labs/eigenda/pull/1368/commits/a01d8fedb7c0a0abcd6b32a4e2aef5c3df7b4912)